### DISCUSSION TITLE
Stop codon found

### DISCUSSION THREAD

**Discussion by linuxbioinfoguy** [liulab-dfci/TRUST4#100] [Q&A]
Hi, I am a beginner in using this package, just take a try, so in the final report tsv file, there are multiple result from the same asembleid, but with slightly different in the nt and the aa, such as following:
31      6.609808e-02    TGTCAACAGAGTTACAGTACCCCGTACACTTTT       CQQSYSTPYTF     IGKV1-39*01     .       IGKJ2*01        IGKC    assemble0       0
12      2.618337e-02    TGTCAATAGAGTTACAGTACCCCGTACACTTTT       CQ_SYSTPYTF     IGKV1-39*01     .       IGKJ2*01        IGKC    assemble0       0
9       2.072495e-02    TGTCAACAGAGTTACAGTACCCCGTATACTTTT       CQQSYSTPYTF     IGKV1-39*01     .       IGKJ2*01        IGKC    assemble0       0

as see, there is a stop codon TAG in the second sequence and show as CQ_SYSTPYTF. Could this kind of result be considered as false? since quite unlikely a valid sequence contains a stop codon at this part, thanks!

**Comment by mourisl** [comment]
Since there are multiple reads supporting this stop codon, I believe this is real. This could be introduced by somatic hypermutations resulting in a non-productive receptor. You may filter those non-productive or keep them depending on your analysis.

Each assemble_id encodes highly similar CDR3s (most likely from somatic hypermutation), and this is what is observed here.

**Reply by linuxbioinfoguy** [reply]
Thanks!
